### ISSUE TITLE
Firefox is not its latest version.

### ISSUE THREAD

**Issue by roke-julian-lockhart** [UWP-Open-Source-Community/Firefox#1]

**Comment by mediaexplorer74** [comment]
> Firefox is not its latest version.

Yes. Can you assemble latest version of GeckoFX engine to build the new Firefox Browser UWP?

**Comment by roke-julian-lockhart** [comment]
@mediaexplorer74, I don't know what that is, nor how it's relevant. This isn't even necessary anymore, though, since [Mozilla provides their own MSIX package](https://firefox-source-docs.mozilla.org/browser/installer/windows/installer/MSIX.html).

**Comment by mediaexplorer74** [comment]
> @mediaexplorer74, I don't know what that is, nor how it's relevant. This isn't even necessary anymore, though, since [Mozilla provides their own MSIX package](https://firefox-source-docs.mozilla.org/browser/installer/windows/installer/MSIX.html).

[Roke Julian Lockhart](https://github.com/rokejulianlockhart/rokejulianlockhart),
ok (got it). IMHO, the "secret learning" (complex of skills such as C++, Python, Rust...) how to assemble (build) the Firefox sources is important , sometimes... For example, over 5 years there are many retrospective Windows10Mobile os's fanboys who cannot do normal websurfing (i.e., opening trello, webwhatsapp, etc.) .
 And I am one of them... "Last of Magicans" :)
 
 P.S. Hehe, the link you provided is... nothing. Mozilla NEVER provide their own APPX package for ARM32 architecture.

**Comment by roke-julian-lockhart** [comment]
> Mozilla NEVER provide their own APPX package

Yes, not an APPX package. MSIX supersedes APPX, right?

> for ARM32 architecture.

Yeah, seems like only ARM64 is supported currently.
